subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformIntegrationTest.groovy

```diff
@@ -822,6 +822,155 @@ class FileSizer extends ArtifactTransform {
         output.count("Transforming") == 0
     }
 
+    def "transform with two attributes will not confuse"() {
```

Comment:
Could you move this test to `DisambiguateArtifactTransformIntegrationTest` which was introduced since your PR was created?